query/src/main/java/tech/ydb/query/QueryTransaction.java

```diff
@@ -0,0 +1,113 @@
+package tech.ydb.query;
+
+import java.util.concurrent.CompletableFuture;
+
+import tech.ydb.core.Result;
+import tech.ydb.core.Status;
+import tech.ydb.query.result.QueryInfo;
+import tech.ydb.query.settings.CommitTransactionSettings;
+import tech.ydb.query.settings.ExecuteQuerySettings;
+import tech.ydb.query.settings.RollbackTransactionSettings;
+import tech.ydb.table.query.Params;
+
+/**
+ * Short-living object allows transactional execution of several queries in one interactive transaction.
+ * @author Aleksandr Gorshenin
+ */
+public interface QueryTransaction {
+
+    /**
+     * Returns identifier of the transaction or null if the transaction is not active (not started/committed/rollbacked)
```

Comment:
Yes. It is very important because the QueryTransaction can be used in implicit mode - without calling commit()/rollback(). When QueryTransaction is not active - any execution of query with commitAtEnd=false starts a new transaction. And execution of query with commitAtEnd=true commits this transaction. So the one QueryTransaction can have different identifiers in different times